### DISCUSSION TITLE
Why are we building nightly, even when there aren't new commits?

### DISCUSSION THREAD

**Discussion by mgrojo** [sqlitebrowser/sqlitebrowser#3810] [General]
It seems a waste of energy and CO<sub>2</sub> emissions. Moreover, it is also a waste of human attention for all of us watching the publication of releases. If there is no good reason to do it nightly, I'd suggest doing only continuous releases, or to do nightly releases only on days which have received new commits.

**Comment by lucydodo** [comment]
I agree, and if others agree as well, I will make the change.

**Reply by justinclift** [reply]
Yeah, I was meaning to ask about this a while ago too.  The `continuous` releases seem like the better thing these days. :smile:

**Reply by lucydodo** [reply]
Changed via 1048982fd40214015f2715031594856306458e6c

**Reply by lucydodo** [reply]
Proof:
- Attempt 1: Run the workflow (https://github.com/sqlitebrowser/sqlitebrowser/actions/runs/12110755139/attempts/1)
- Attempt 2: Skip the workflow (https://github.com/sqlitebrowser/sqlitebrowser/actions/runs/12110755139/attempts/2)

**Reply by justinclift** [reply]
Awesome, good stuff. :smile:

**Reply by juhalipponen** [reply]
Looks like nightly build is broken or at least the distribution system is broken, Looking at :
https://nightlies.sqlitebrowser.org/latest/
The reported file sizes are few tens of bytes, and the files cannot be downloaded.

Maybe this problem is caused by the above change?
Are historical nighly builds stored anywhere?

I would need a version with support for STRICT tables, seems that it's still not in 
the release.

**Reply by lucydodo** [reply]
Yes, you're right. I forgot to modify the script that does that as well.
Now I fixed this issue with 7b51cb5c207bc69c7444e0833324a11a7c55c6e4 commit, Can you please check again?

**Reply by justinclift** [reply]
> Are historical nighly builds stored anywhere?

@juhalipponen Yep.  They're on https://nightlies.sqlitebrowser.org under OS specific directories. ie `win32`, `win64`, `macos-universal`

Hmmm, we don't seem to store the AppImages there though.  @lucydodo Any idea if they'd be easy to add?

**Reply by lucydodo** [reply]
Good idea. I just added AppImage builds to the nightly build CI: 22ecfe371f37910d8271ef9452d94ec9f89e37f4
